test/hotspot/gtest/runtime/test_os.cpp

```diff
@@ -258,7 +258,7 @@ TEST_VM(os, test_print_hex_dump) {
 
   // two pages, first one protected.
   const size_t ps = os::vm_page_size();
-  char* two_pages = os::reserve_memory(ps * 2, false, mtTest);
+  char* two_pages = os::reserve_memory(ps * 2, mtTest, false);
```

Comment:
```suggestion
  char* two_pages = os::reserve_memory(ps * 2, mtTest);
```

Reply:
ExecMem is a parameter, that could be false.